warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/partition/MultiTableRangePartitioner.java

```diff
@@ -58,15 +60,16 @@ private void readCacheFilesIfNecessary() {
                 // We need a replacement that isn't deprecated, but context.getCacheFiles() returns paths that are not local
                 // No Hadoop documentation seems to indicate what is the correct replacement for this method
                 localCacheFiles = context.getLocalCacheFiles();
-            } catch (IOException e) {
+            } catch (Exception e) {
                 log.error("Failed to get localCacheFiles from context", e);
                 throw new RuntimeException("Failed to get localCacheFiles from context", e);
             }
             
             try {
-                NonShardedSplitsFile.Reader reader = new NonShardedSplitsFile.Reader(context.getConfiguration(), localCacheFiles, isTrimmed());
+                NonShardedSplitsFile.Reader reader = new NonShardedSplitsFile.Reader(context.getConfiguration(), localCacheFiles, getSplitsFileType());
+                splitToLocationMap.set(reader.getSplitsAndLocationsByTable());
```

Comment:
given the splits file type, can't we determine whether we should even try to call getSplitsAndLocations